src/pages/UserProfile/ProfileDetails.jsx

```diff
@@ -1,102 +1,160 @@
 import { Link } from "react-router-dom";
+import { profileDetails } from "./ProfileDetails.module.css";
 
 export const ProfileDetails = ({ user, userId }) => {
-  const { username, password } = user.account;
-  const { firstName, lastName, birthDate, email, address } = user.personalInfo;
-  const { company, fax, facebook, phones } = user.contactDetails;
-  const { skills, hobbies } = user.capabilities;
+  const {
+    username,
+    password,
+    firstName,
+    lastName,
+    birthDate,
+    email,
+    address,
+    company,
+    fax,
+    facebook,
+    skills,
+    hobbies,
+  } = user;
+
+  const phones = [];
+  for (let i = 1; i <= 3; i++) {
```

Comment:
avoid using legacy loops. `for of`, `for in`, `forEach` are good ES6 loop iterators